### DISCUSSION TITLE
2 Crazyflies with Loco Pos. Sys.

### DISCUSSION THREAD

**Discussion by PumaRub** [bitcraze/discussions#101] [Q&A]
Hey all!,

i wanted to fly two Crazyflie-drones via Loco Pos. Sys. and for that i just wanted to use the example file called: swarmSequence.py.
My Loco. Pos. System works fine and i already tested it with also a example file called: initial_position.py . The Positions i enter to the  swarmSequence.py - file are not approached, instead the drones fly somewhere else. 
Am i doing something wrong?
Or is there any other example file for my intention (Flying 2 Drones to given Points with Loc. Pos. Sys.) 

thank you,

yours Pumarub

**Comment by knmcguire** [comment]
Hi! Some questions first:

- Could you share your code you are using?
- Have you tried using the script without using 'initial_position.py' first? 
 
That script (initial_position.py) can do some weird stuff with the positioning... it gives an different initial position than the one that you have configured it too so I don't think you should try it out for testing. Probably /autonomy/autonomousSequence.py is better to test with a single drone.
